### ISSUE TITLE
Data import - option to ignore quotes

### ISSUE THREAD

**Issue by mkutmon** [PathVisio/pathvisio#29]
Reported by Lars:
Pathvisio does not seem to be able to import quoted text files, for example:

"ID" "FC" "P"

"gene1" 2.23 0.05

"gene2" 1.23 0.01

(where the separator was a tab in my case).

Especially when values in a certain column contain special characters or spaces, many tools automatically add quotes to the fields. Of course, one can remove the quotes from the input file, but people using Excel or any spreadsheet software as a viewer will generally not be aware that they are there. So it may be helpful to allow import of such files. Now, import runs without errors, but the numeric data are not recognized as being numeric.

**Comment by DeniseSl22** [comment]
Yes, this would help users indeed. I'll add it to the list of PV4.

**Comment by traybug23** [comment]
Hi @mkutmon @DeniseSl22  ! I'm interested in devising a fix for this issue. Are there any additional pointers or informations I need to know before I carry forward?

**Comment by DeniseSl22** [comment]
Hi @traybug23 : it might be nice to create a mock dataset to test if this behaviour still exists in PathVisio 4. I hope the short example above helps with creating this dataset.

**Comment by traybug23** [comment]
> Hi [@traybug23](https://github.com/traybug23) : it might be nice to create a mock dataset to test if this behaviour still exists in PathVisio 4. I hope the short example above helps with creating this dataset.

I tried reproducing this error as you mentioned but i keep running into an error where the import data wizard does not show up for me. 
I went through this tutorial on PathVisio to find out how exactly we can load the expression datasets in text files but in PathVisio 4 I am unable to see this 'import expression data' window as mentioned in the tutorial. 

 If you can provide me with any guidance on how this feature exists in PV4 or if it requires to download any particular plugin for it, I would be very grateful.

<img width="941" height="925" alt="Image" src="https://github.com/user-attachments/assets/6279e14f-bdf4-4df7-9d22-d034ff0a2c29" />
<img width="1916" height="1141" alt="Image" src="https://github.com/user-attachments/assets/f717a5b1-9b65-40c0-8688-e9bcdb88a4b5" />

**Comment by DeniseSl22** [comment]
Actually now thinking about it, in PV4 we are probably not including this option anymore.... I'll have to check with @mkutmon  on this though; sorry for sending you on this goose chase... 

Might I in the meantime suggest this issue for example: https://github.com/PathVisio/pathvisio/issues/221 ?

**Comment by traybug23** [comment]
No worries, I will look into #221 on adding a redo button to PathVisio, I have personally faced that same issue too with PathVisio while building a pathway. Can I also suggest on adding an eraser button where incorrectly placed datanodes, interaction arrows etc can be erased. Thanks a lot for letting me know about this!
